### PR TITLE
Remove api-doc generation from appcenter-docs

### PR DESCRIPTION
**Pull request by lauraviles**

### CODE REVIEW

File: venv.sh [hunk 1]
```diff
@@ -72,12 +72,10 @@ function copy-uforge-api() {
```
Comment (cinlloc) [inline venv.sh]: Lines above (from 51 to 71) are about API doc also

File: venv.sh [hunk 2]
```diff
@@ -86,7 +84,7 @@ echo -e "  copy-uforge-api ../uforge"
 echo -e "  \e[2m# the above is the default value\e[0m"
 echo -e "  copy-uforge-api"
 echo -e
```
Comment (cinlloc) [inline venv.sh]: Lines from 79 to 87 are about API doc also

File: venv.sh (hunk 1 above)
Comment (lauraviles) [reply venv.sh]: Done

File: venv.sh (hunk 2 above)
Comment (lauraviles) [reply venv.sh]: Done

**Comment by cinlloc** [comment]
Replaced by #1014

**Pull request closed without merging**